src/app/catalog/components/learningpath-catalog/learningpath-catalog.component.ts

```diff
@@ -105,6 +115,26 @@ export class LearningPathsCatalogComponent extends BaseRoutableComponent impleme
 		return (badge) => (tag ? badge.tags.includes(tag) : true);
 	}
 
+	prepareTexts() {
+		this.plural = {
+			issuer: {
+				'=0': this.translate.instant('Issuer.noInstitutions'),
+				'=1': '1 Institution',
+				other: '# ' + this.translate.instant('General.institutions'),
+			},
+			issuerText: {
+				'=0': this.translate.instant('Issuer.institutionsIssued'),
+				'=1': '1 ' + this.translate.instant('Issuer.institutionIssued'),
+				other: '# ' + this.translate.instant('Issuer.institutionsIssued'),
+			},
+			learningPath: {
+				'=0': this.translate.instant('General.noLearningPaths'),
```

Comment:
Should this show `noLearningPaths` text?
<img width="879" alt="Screenshot 2024-11-25 at 11 24 35" src="https://github.com/user-attachments/assets/cdd3c16b-6cc9-4519-b16e-de9ec953b433">